inductiva/fluids/post_processing/openfoam.py

```diff
@@ -0,0 +1,372 @@
+"""Visualization processing of steady-state simulations with OpenFOAM.
```

Comment:
All of the post-processing files need a slight refactor, since currently it is a mess and no structure follows.

I will make the doc string cleaner.